src/proto/parameters.proto

```diff
@@ -658,4 +658,16 @@ message DefenderStealConfig
     // Max speed of ball that pass defender will try and get possession
     required double max_ball_speed_to_get_m_per_s = 3
         [default = 0.5, (bounds).min_double_value = 0.0, (bounds).max_double_value = 7.0];
+    // The additional buffer length for each side of the goal
+    // to determine if crease defender chipping is "towards goal"
+    required double goal_post_offset_chipping = 4
+        [default = 0.5, (bounds).min_double_value = 0.0, (bounds).max_double_value = 2.0];
```

Comment:
This is already contained within `CreaseDefenderConfig`. Pass defenders do not have chipping actions, why is this included here?

Reply:
oh that must have been added in the git merge - good catch.